### ISSUE TITLE
AccessibilityInfo Module

### ISSUE THREAD

**Issue by baarkerlounger** [status-im/react-native-desktop-qt#593]
## Environment
```
System:
    OS: macOS 10.15.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 429.95 MB / 16.00 GB
    Shell: 5.0.17 - /usr/local/bin/bash
  Binaries:
    Node: 14.5.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.5 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.3 - /Users/.rvm/rubies/ruby-2.7.1/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK: Not Found
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5692245
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Java: Not Found
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0 
    react-native: 0.62.2 => 0.62.2 
  npmGlobalPackages:
    *react-native*: Not Found
```

## Description
```
error: Error: Unable to resolve module `./Libraries/Components/AccessibilityInfo/AccessibilityInfo` from `node_modules/react-native/index.js`: 

None of these files exist:
  * node_modules/react-native/Libraries/Components/AccessibilityInfo/AccessibilityInfo(.native|.desktop-qt.js|.native.js|.js|.desktop-qt.json|.native.json|.json|.desktop-qt.ts|.native.ts|.ts|.desktop-qt.tsx|.native.tsx|.tsx)
  * node_modules/react-native/Libraries/Components/AccessibilityInfo/AccessibilityInfo/index(.native|.desktop-qt.js|.native.js|.js|.desktop-qt.json|.native.json|.json|.desktop-qt.ts|.native.ts|.ts|.desktop-qt.tsx|.native.tsx|.tsx)
    at ModuleResolver.resolveDependency (/Users/Documents/mobile/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:163:15)
    at ResolutionRequest.resolveDependency (/Users/Documents/mobile/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/Users/Documents/mobile/node_modules/metro/src/node-haste/DependencyGraph.js:287:16)
    at Object.resolve (/Users/Documents/mobile/node_modules/metro/src/lib/transformHelpers.js:267:42)
    at /Users/Documents/mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:434:31
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/Documents/mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:431:18)
    at /Users/Documents/mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:275:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/Documents/mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
```

## Reproducible Demo
Trying to get this project running https://github.com/standardnotes/mobile. It successfully builds using `npx  react-native run-desktop` but  crashes on startup:

```
JS server already running.
Starting the app...
ViewManager: Created view:  ReactActivityIndicator_QMLTYPE_4(0x7fdcbb792da0, parent=0x0, geometry=0,0 12x12) . Source QML file:  "qrc:/qml/ReactActivityIndicator.qml" . Props keys:  ()
ViewManager: Created view:  ReactButton_QMLTYPE_6(0x7fdcbd28e370, parent=0x0, geometry=0,0 16x12) . Source QML file:  "qrc:/qml/ReactButton.qml" . Props keys:  ()
ViewManager: Created view:  ReactImage_QMLTYPE_17(0x7fdcc3f0fd90, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactImage.qml" . Props keys:  ()
ViewManager: Created view:  ReactModal_QMLTYPE_18(0x7fdcc3f32c40, name="modalRoot", parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactModal.qml" . Props keys:  ()
ViewManager: Created view:  ReactNavigator_QMLTYPE_19(0x7fdcbb79c310, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactNavigator.qml" . Props keys:  ()
ViewManager: Created view:  ReactPicker_QMLTYPE_21(0x7fdcbb7a6aa0, parent=0x0, geometry=0,0 60x200) . Source QML file:  "qrc:/qml/ReactPicker.qml" . Props keys:  ()
ViewManager: Created view:  ReactRawText_QMLTYPE_22(0x7fdcbb698690, parent=0x0, geometry=0,0 0x16) . Source QML file:  "qrc:/qml/ReactRawText.qml" . Props keys:  ()
ViewManager: Created view:  ReactScrollView_QMLTYPE_23(0x7fdcbb6a2ce0, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactScrollView.qml" . Props keys:  ()
ViewManager: Created view:  ReactSlider_QMLTYPE_24(0x7fdcbb6b52d0, parent=0x0, geometry=0,0 200x40) . Source QML file:  "qrc:/qml/ReactSlider.qml" . Props keys:  ()
ViewManager: Created view:  ReactSwitch_QMLTYPE_26(0x7fdcbb6c55d0, parent=0x0, geometry=0,0 74x40) . Source QML file:  "qrc:/qml/ReactSwitch.qml" . Props keys:  ()
ViewManager: Created view:  ReactText_QMLTYPE_28(0x7fdcbd0556b0, parent=0x0, geometry=0,0 0x16) . Source QML file:  "qrc:/qml/ReactText.qml" . Props keys:  ()
ViewManager: Created view:  ReactTextInput_QMLTYPE_29(0x7fdcbd294860, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactTextInput.qml" . Props keys:  ()
ViewManager: Created view:  ReactView_QMLTYPE_31(0x7fdcbb7da7c0, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactView.qml" . Props keys:  ()
ViewManager: Created view:  ReactWebView_QMLTYPE_32(0x7fdcbb7e4e80, parent=0x0, geometry=0,0 0x0) . Source QML file:  "qrc:/qml/ReactWebView.qml" . Props keys:  ()
auto SourceCode::loadSource(QNetworkAccessManager *)::(anonymous class)::operator()() const : Error while loading source "Error transferring http://localhost:8081/index.bundle?platform=desktop-qt&dev=true - server replied: Internal Server Error"
``` 

Am I right in thinking the AccessibilityInfo is an unsupported module?
